pkgs/top-level/all-packages.nix

```diff
@@ -16823,8 +16823,9 @@ in
 
   libutempter = callPackage ../development/libraries/libutempter { };
 
-  libunwind = if stdenv.isDarwin
-    then darwin.libunwind
+  libunwind =
+    if stdenv.isDarwin then darwin.libunwind
+    else if stdenv.hostPlatform.isRiscV then llvmPackages_latest.libunwind
```

Comment:
What is the default LLVM for riscv? is it also the latest?

Reply:
Currently it's [set to 9](https://github.com/NixOS/nixpkgs/blob/4df31757808eeebe5761bdc80481d0c0a473b4cf/pkgs/top-level/all-packages.nix#L11460) because it's the lowest LLVM version with riscv64 officially supported. However, LLVM libunwind did not gain support for RISC-V until LLVM 10.